Site/SiteAnalyticsModule.php

```diff
@@ -503,6 +643,21 @@ protected function getFacebookPixelCommand($command)
 	}
 
 	// }}}
+	// {{{ protected function getBingUETCommand()
+
+	protected function getBingUETCommand($command)
+	{
+		if (!is_array($command)) {
+			$command = array($command);
+		}
+
+		return sprintf(
+			'window.uetq.push(%s);',
+			implode(', ', array_map('json_encode', $command))
```

Comment:
Is this supposed to be done in one push command? If so we need to rewrite this method so that it isn't called in a for loop.